src/styles/theme.js

```diff
@@ -2,22 +2,27 @@
 
 import shadows from './shadows';
 import transitions from './transitions';
-import createTypography from './typography';
-import createBreakpoints from './breakpoints';
-import createPalette from './palette';
+import createTypography from './createTypography';
+import createBreakpoints from './createBreakpoints';
+import createPalette from './createPalette';
+import createMixins from './createMixins';
 import zIndex from './zIndex';
-import createMixins from './mixins';
 import spacing from './spacing';
 
 function createMuiTheme(options: Object = {}) {
```

Comment:
Thanks for the file renames, it's been bothering me but I haven't addressed it.  This file could also be renamed.  I'd prefer `createTheme.js` with an `export createTheme` instead of `createMuiTheme`.  The `Mui` is redundant since we are already namespaced.

Reply:
Oh boy. I wasn't sure about this one! Ok, I guess we could go into this direction. `theme` can be confusing!

Reply:
done